yaspin/core.py

```diff
@@ -379,6 +398,11 @@ def _compose_out(self, frame, mode=None):
         if self._side == "right":
             frame, text = text, frame
 
+        if self._timer:
+            text += " ({})".format(
+                datetime.timedelta(seconds=int(self.elapsed_time))
```

Comment:
Wrapping `self.elapsed_time` by `int` results in dropping microseconds data. This may be important for operations running less than 1 second. I'd rather remove `int`, but this results in strings with 6-digit microseconds data, which I believe is too long:

```
> str(datetime.timedelta(seconds=elapsed_time))
'0:00:06.519013'
```

Let's persist microseconds data and find a way to truncate it to 2 digits.

Reply:
Great suggestion, done